### PR TITLE
Migrate raised prop back to Button component

### PR DESCRIPTION
**Pull request by whitedogg13**
This PR tries to fix issue #1028 to bring `raised` prop back to `Button` component

### CODE REVIEW

**Comment by iRoachie** [comment]
Firstly, let's add this new prop to the typescript definitions. You can add it anywhere inside this interface https://github.com/react-native-training/react-native-elements/blob/master/src/index.d.ts#L218. 

Secondly, you'll need to add it to the docs. The file to edit is here https://github.com/react-native-training/react-native-elements/blob/master/docs/button.md#props. You can look at the other props in the doc to see the pattern of how we write them.

if you need any help doing any of these things, you can let me know.

**Comment by whitedogg13** [comment]
@iRoachie Roger that! The doc and typescript definition has been updated, thanks for helping me!

File: docs/button.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ import Icon from 'react-native-vector-icons/FontAwesome';
 * [`disabled`](#disabled)
 * [`disabledStyle`](#disabledstyle)
 * [`disabledTitleStyle`](#disabledtitlestyle)
+* [`raised`](#raised)
```
Comment (iRoachie) [inline docs/button.md]: Can you just adjust the order here? props are alphabetically listed.

**Comment by iRoachie** [comment]
Alright awesome, you just need to update your snapshots now. `npm run test -- -u`

**Comment by whitedogg13** [comment]
@iRoachie Thanks for helping me 👍

**Comment by iRoachie** [comment]
Fantastic job man!

**Pull request merged**